### DISCUSSION TITLE
Linux AppImage not able to stay logged in:

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [tutao/tutanota#6916] [Q&A]
Hello,

I have the latest linux appimage and when I try to log in it says connection to the server was lost, please try again. If I try again it almost logs in then fails again with the same error message and if I try yet again it sometimes just returns the original server lost message? I assume it may be my Linux distro causing this? I can however log into my email account using my web browser without a problem. Any ideas would be helpful. Thank you :)

**Comment by charlag** [comment]
Hi, sorry to hear about this, it might be this issue: https://github.com/tutao/tutanota/issues/6641

**Reply by BonzoDogDooDah** [reply]
Hi I will try this and then let you know but I see a lot of users saying the same as me so will this issue be fixed in a future update? :)

**Reply by Opening-Button-8988** [reply]
I'd like to point out that I still have this issue. IPv6 is apparently required which is stupid. Please update the app to be functional without the requirement of the IPv6 protocol. IPv6 is worse for privacy, especially when you use a VPN due to possible traffic leakage.
